contracts/LivepeerProtocol.sol

```diff
@@ -121,6 +121,22 @@ contract LivepeerProtocol {
     // rewardMultiplier[1] -> transcoder's cumulative stake for round
     mapping (address => mapping (uint256 => uint256[2])) public rewardMultiplierPerTranscoderAndRound;
 
+    // The various states a job can be in
+    enum JobStatus { Inactive, Active }
```

Comment:
As mentioned in Doug's previous comment, a transcoder might submit transcode claims for multiple ranges of segments. As a result, I don't think we can determine on-chain whether all the work for a job has been claimed without additional information. Furthermore, since multiple submissions of transcode claims will require multiple calls to `verify()`, I also don't think we can determine on-chain whether all the claims for a job have been verified without additional information.

It seems that as of right now we can only accurately say that a job is one of two states: `Inactive` or `Active`. A job would become `Active` when a broadcaster submits a job using `job()`. A job would become `Inactive` after either the broadcaster or transcoder for a job calls `endJob()` and a certain number of blocks has passed.